public/index.html

```diff
@@ -17,7 +17,7 @@
       socket.io events emitted from the server.  It is used to
       proactively terminate sessions when the session timeout expires.
     -->
-    <script src="https://localhost/socket.io/socket.io.js"></script>
+    <!--<script src="https://localhost/socket.io/socket.io.js"></script>-->
```

Comment:
Is this no longer needed because you added `socket.io-client` as a dependency in package.json?